cli/cmd/deploy.go

```diff
@@ -241,16 +241,22 @@ func didAllResultsError(results []schema.DeployResult) bool {
 	return true
 }
 
-func getAPICommandsMessage(results []schema.DeployResult) string {
+func getAPICommandsMessage(results []schema.DeployResult, envName string) string {
 	apiName := "<api_name>"
 	if len(results) == 1 {
 		apiName = results[0].API.Name
 	}
 
+	defaultEnv := getDefaultEnv(_generalCommandType)
+	var envArg string
+	if envName != defaultEnv {
+		envArg = " --env " + envName
```

Comment:
It may be easier to read the code below if the leading space was removed e.g. `envArg = "--env " + envName`